ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -647,23 +644,6 @@ public void shouldFailWhenAvroSchemaCanNotBeEvolved() {
         containsString("Cannot register avro schema for S1 as the schema registry rejected it"));
   }
 
-  @Test
-  public void shouldDistributeMultipleInterDependantStatements() {
```

Comment:
This test requires to much plumbing to do in a unit test - so this has been moved to the new functional test.